packages/desktop-client/src/components/accounts/Reconcile.jsx

```diff
@@ -59,33 +60,42 @@ export function ReconcilingMessage({
                 marginRight: 3,
               }}
             />
-            All reconciled!
+            <Trans>All reconciled!</Trans>
           </View>
         ) : (
           <View style={{ color: theme.tableText }}>
             <Text style={{ fontStyle: 'italic', textAlign: 'center' }}>
-              Your cleared balance{' '}
-              <strong>{format(cleared, 'financial')}</strong> needs{' '}
-              <strong>
-                {(targetDiff > 0 ? '+' : '') + format(targetDiff, 'financial')}
-              </strong>{' '}
-              to match
-              <br /> your bank’s balance of{' '}
-              <Text style={{ fontWeight: 700 }}>
-                {format(targetBalance, 'financial')}
-              </Text>
+              <Trans>
+                Your cleared balance{' '}
+                <strong>
+                  {{ clearedBalance: format(cleared, 'financial') }}
+                </strong>{' '}
+                needs{' '}
+                <strong>
+                  {{
+                    difference:
+                      (targetDiff > 0 ? '+' : '') +
+                      format(targetDiff, 'financial'),
+                  }}
+                </strong>{' '}
+                to match
+                <br /> your bank&aposâs balance of{' '}
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏what's this?

```suggestion
                <br /> your bank's balance of{' '}
```

Reply:
@MatissJanis I think it was a weird typo that already existed.  It is supposed to be "bank's" but somehow got converted into an HTML entity `&apos;` and then the semi was converted into an `â` somehow.